game/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "cribbage-game"
+version = "0.1.0"
+edition = "2018"
+
+[dependencies]
+strum = "0.21"
+strum_macros = "0.21"
+arrayvec="0.7.1"
+rand= "0.8.4"
+serde = { version = "1.0.130", features = ["derive"] }
+serde_json= "1.0.67"
+itertools= "0.10"
```

Comment:
We should replace the last  use of IterTools with the combinator module.

While we're at it, we can clean up combinator in this repo. It's easy to make it clippy clean:
https://github.com/DanMass/fantastic-barnacle/blob/main/src/combinator.rs